OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -0,0 +1,391 @@
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane4.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void pictureBoxPane5_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane5.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane5.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane5.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void pictureBoxPane6_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane6.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane6.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane6.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        private void checkBoxGuassianBlur_CheckedChanged(object sender, EventArgs e)
+        {
+            try
+            {
+                KeyValPair keyValuePair = new KeyValPair();
+                keyValuePair = getMeterKeyValuePair();
+                if (checkBoxGuassianBlur.Checked)
+                {
+                    
+                    Cv2.GaussianBlur(keyValuePair.meter.SourceImage, keyValuePair.meter.ModifiedImage, new OpenCvSharp.Size((int)numericUpDown1.Value, (int)numericUpDown2.Value), 0, 1, BorderTypes.Default);
+                    UpdateModifiedImage(keyValuePair);
+                    DemoUtilities.loadImage(GetSelectedPictureBox(), keyValuePair.meter.ModifiedImage);
+                }
+                else
+                {
+                    DemoUtilities.loadImage(GetSelectedPictureBox(), keyValuePair.meter.SourceImage);
```

Comment:
I've the modified image is the same type as the source image, then you can consolidate the `if...else`. Something like:

```c#
var image = keyValuePair.meter.SourceImage;
if (checkBoxGuassianBlur.Checked) {
   ...
   image = keyValuePair.meter.ModifiedImage;
}
DemoUtilities.loadImage(GetSelectedPictureBox(), image);;
```